react-app/src/components/ConnectWalletModal/ConnectWalletModal.tsx

```diff
@@ -0,0 +1,191 @@
+          leaveFrom="opacity-100"
+          leaveTo="opacity-0"
+        >
+          <div className="fixed inset-0 bg-black/30" />
+        </Transition.Child>
+
+        <div
+          className={cn("fixed", "inset-0", "overflow-y-auto", "select-none")}
+        >
+          <div
+            className={cn(
+              "flex",
+              "min-h-full",
+              "items-center",
+              "justify-center",
+              "text-center",
+              "px-4"
+            )}
+          >
+            <Transition.Child
+              as={Fragment}
+              enter="ease-out duration-300"
+              enterFrom="opacity-0 scale-95"
+              enterTo="opacity-100 scale-100"
+              leave="ease-in duration-200"
+              leaveFrom="opacity-100 scale-100"
+              leaveTo="opacity-0 scale-95"
+            >
+              <Dialog.Panel
+                className={cn(
+                  "w-full",
+                  "max-w-sm",
+                  "transform",
+                  "overflow-hidden",
+                  "rounded-lg",
+                  "bg-white",
+                  "p-6",
+                  "shadow-xl",
+                  "transition-all",
+                  "flex",
+                  "flex-col",
+                  "gap-y-2"
+                )}
+              >
+                <Dialog.Title
+                  className={cn(
+                    "text-lg",
+                    "leading-6",
+                    "font-medium",
+                    "text-gray-900"
+                  )}
+                >
+                  <LocalizedText messageID="WalletConnect.connectModal.title" />
+                </Dialog.Title>
+                <Dialog.Description
+                  className={cn(
+                    "text-sm",
+                    "leading-5",
+                    "font-normal",
+                    "text-gray-500"
+                  )}
+                >
+                  <LocalizedText messageID="WalletConnect.connectModal.description" />
+                </Dialog.Description>
+                <div className={cn("flex", "flex-col", "mt-3")}>
+                  <button
+                    className={cn(
+                      "flex",
+                      "flex-row",
+                      "items-center",
+                      "text-left",
+                      "p-4",
+                      "gap-x-2"
+                    )}
+                    type="button"
+                    onClick={onKeplrConnect}
+                  >
+                    <KeplrLogo height={40} width={40} />
+                    <p
+                      className={cn(
+                        "flex-1",
+                        "text-sm",
+                        "leading-5",
+                        "font-medium",
+                        "text-likecoin-green"
+                      )}
+                    >
+                      <LocalizedText messageID="WalletConnect.connectModal.keplrWallet" />
```

Comment:
@hochiw can you help rename the strings as well? 🙏

Reply:
oh right